RockWeb/Themes/NewSpring/assets/Lava/LAYOUT/collection-detail.lava

```diff
@@ -13,94 +13,66 @@
-    {% if baseBackgroundColor and baseBackgroundColor != empty %}
-        {% assign backgroundColor = baseBackgroundColor %}
+    {% if Item.BackgroundColor != empty %}
         <style>
             .brand-bg {
-                background-color: {{ backgroundColor | Darken:'10%' }};
+                background-color: {{ Item.BackgroundColor | Prepend:'#' }};
             }
         </style>
     {% endif %}
 
 
     <!-- SIDE BY SIDE -->
-    {% capture id %}{% endcapture %}
-    {% capture cciid %}{{ Item.Id }}{% endcapture %}
-    {% capture guid %}{{ Item.Guid }}{% endcapture %}
-    {% capture title %}{{ Item.Title | Replace:"'","’" }}{% endcapture %}
-    {% capture content %}{{ Item | Attribute:'Summary' | Replace:"'","’" }}{% endcapture %}
-    {% capture textalignment %}{% endcapture %}
-    {% capture type %}{% endcapture %}
-    {% capture label %}{{ type }}{% endcapture %}
-    
-    {% assign isDateVisible = Item.ContentChannel | Attribute:'IsDateVisible','RawValue' %}
-    {% if isDateVisible == 'True' %}
-        {% assign dates = Item | Attribute:'Dates','RawValue' %}
-        {% assign dateParts = dates | Split:',' %}
-        {% assign datePartsSize = dateParts | Size %}
-        {% assign datePart1 = dateParts[0] | Date:'yyyyMMdd' %}
-        {% if datePartsSize > 1 %}
-            {% assign datePart2 = dateParts[1] | Date:'yyyyMMdd' %}
-        {% endif %}
-        {% if dates != empty %}
-            {% capture subtitle %}{[ formatDate date:'{{ dateParts[0] }}' ]}{% if datePartsSize > 1 and datePart1 != datePart2 %} - {[ formatDate date:'{{ dateParts[1] }}' ]}{% endif %}{% endcapture %}
+    {% assign id = Item.Id %}
+    {% assign cciid = Item.Id %}
+    {% assign dataset = Item.Dataset %}
+    {% assign guid = Item.Guid %}
+    {% assign title = Item.Title | Replace:"'","’" %}
+    {% assign content = Item.Summary | HtmlDecode | Replace:"'","’" %}
+
+    {% assign isDateVisible = Item.IsDateVisible %}
+    {% if isDateVisible == 'Yes' %}
+        {% if Item.StartDateTime != empty %}
+            {% capture subtitle %}{[ formatDate date:'{{ Item.StartDateTime }}' ]}{% if Item.EndDateTime != empty and Item.StartDateTime != Item.EndDateTime %} - {[ formatDate date:'{{ Item.EndDateTime }}' ]}{% endif %}{% endcapture %}
         {% endif %}
     {% endif %}
 
-    {% capture imageurl %}{{ Item | Attribute:'ImageSquare','Url' }}{% endcapture %}
-    {% capture imageoverlayurl %}{% endcapture %}
+    {% capture imageurl %}{{ Item.ImageSquare }}{% endcapture %}
     {% capture imagealignment %}Left{% endcapture %}
-    {% capture imageopacity %}{% endcapture %}
-    {% capture grayscale %}{% endcapture %}
-    {% capture backgroundvideourl %}{% endcapture %}
-    {% assign downloads = Item | Attribute:'Downloads' %}
-    {% capture lava %}{{ downloads }}{% endcapture %}
-    {% capture video %}{{ Item | Attribute:'Video','RawValue' }}{% endcapture %}
+    {% capture video %}{{ Item.Video }}{% endcapture %}
```

Comment:
This is just one example (there are many from what I can see), but if we're not doing any processing on the item here, do we need to `capture` it? Or `assign` it (as the case may be)? Couldn't we just use the item in the place we need to?

If the point is just so that we have a pretty package at the bottom that we are sending to the next shortcode, I'd say we should change that. This seems like a lot of extra processing across all the lava files just to have the names match up in our parameter list.

Reply:
It's more so the opposite - if those things ever need to change or we're debugging, it's much easier to find/see what's going on if they're separated out instead of all jammed together in a single-line shortcode.

As for capture vs. assign, originally I did them all as capture because some of them had to be, and I was trying be consistent. But we can definitely change the ones that don't need to be captures to assign if you think that'd be better.

Reply:
I don't know if there is a gain to be had by changing `captures` to `assigns`. If there is then yeah, we should do that. 

I was more or less concerned that we were processing a lot of things that didn't need to be processed. Just trying to keep things speedy, right? 

So instead of doing the `{% capture thing%}{{ Item.thing }}{% endcapture %}` here and then passing `thing: {{ thing }}` later, why not just pass `thing: {{ Item.thing }}` and skip the `capture` altogether. 

If you don't feel like we're losing anything by doing it the way we are, then I wouldn't worry about it.